component/registration_form.jsx

```diff
@@ -0,0 +1,7 @@
+export const registration_form = () => (
+  <form method={"post"} action={"/api/user/register_form"}>
+     <input type="text" name="own_contact" placeholder="please enter your phone number" />
+    <input type="text" name="emergency_contact" placeholder="please enter a phone number to join in case of emergency" />
+    <button type={"submit"}> submit </button>
+  </form>
```

Comment:
we are not going to use a form submit, this would create a page reload.
Instead we will manually trigger a fetch query.